packages/tpetra/core/src/Tpetra_Details_computeOffsets.hpp

```diff
@@ -344,6 +351,7 @@ computeOffsetsFromCounts (const OffsetsViewType& ptr,
         Kokkos::parallel_scan (range, functor, total, funcName);
       }
       else { // make tmp copy of counts accessible from offsets' space
+#endif // Tpetra_Workaround5179
         using dev_counts_type = Kokkos::View<count_type*,
           typename CVT::array_layout, device_type>;
         dev_counts_type counts_d ("counts_d", numCounts);
```

Comment:
I'm curious what happens if we allocate `counts_d` uninitialized.  Could it be that some entries of `counts` aren't initialized on entry?
```c++
using Kokkos::view_alloc;
using Kokkos::WithoutInitializing;
dev_counts_type counts_d (view_alloc ("counts_d", WithoutInitializing), numCounts);
```